src/core/common/random_manager.hpp

```diff
@@ -45,6 +45,11 @@
 
 #include "utils/wrap_stdint.h"
 
+#if (!defined(MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES) && \
+     (!defined(MBEDTLS_NO_PLATFORM_ENTROPY) || defined(MBEDTLS_HAVEGE_C) || defined(MBEDTLS_ENTROPY_HARDWARE_ALT)))
+#define MBEDTLS_STRONG_DEFAULT_ENTROPY_PRESENT
```

Comment:
This define appears to be specific to OpenThread. Should we prefix it with `OT_`?